.github/workflows/_build_jax.yaml

```diff
@@ -37,10 +37,6 @@ on:
         description: 'User email in GIT to perform git pull/push'
         required: false
         default: 'jax@nvidia.com'
-      TRIAL_BRANCH:
```

Comment:
Is this duplicating part of #465?

Reply:
Yes, because I started this PR before #465